common/templates/student/edx_ace/passwordreset/email/body.html

```diff
@@ -6,25 +6,42 @@
 <table width="100%" align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
     <tr>
         <td>
-            <h1>
-                {% trans "Password Reset" %}
-            </h1>
-            <p style="color: rgba(0,0,0,.75);">
-                {% blocktrans %}You're receiving this e-mail because you requested a password reset for your user account at {{ platform_name }}.{% endblocktrans %}
-                <br />
-            </p>
-
             {% if failed %}
+                <h1>
+                    {% trans "Password Reset Unsuccessful" %}
+                </h1>
+                <p style="color: rgba(0,0,0,.75);">
+                    {% blocktrans %}You have received this email message because this email address was submitted to {{ platform_name }} for password assistance.{% endblocktrans %}
+                    <br />
+                </p>
+
                 <p style="color: rgba(0,0,0,.75);">
-                    {% blocktrans %}However, there is currently no user account associated with your email address: {{ email_address }}.{% endblocktrans %}
+                    {% blocktrans %}This email address is not associated with an {{ platform_name }} account. As a result, {{ platform_name }} has taken no action and will not use or share this email address.{% endblocktrans %}
                     <br />
                 </p>
 
                 <p style="color: rgba(0,0,0,.75);">
-                    {% trans "If you did not request this change, you can ignore this email." %}
+                    {% blocktrans %}<strong>To reset your {{ platform_name }} account password</strong>, go to the {{ platform_name }} sign-in page, click the <strong>Forgot Password?</strong> link, and enter the email address that is associated with your {{ platform_name }} account.{% endblocktrans %}
                     <br />
                 </p>
+
+                {% if help_link %}
+                    <p style="color: rgba(0,0,0,.75);">
+                            {% blocktrans %}For more information, see this {{ platform_name }} <a href="{{ help_link }}">Help Center article</a>.{% endblocktrans %}
+                        <br />
+                    </p>
+                {% endif %}
+
             {% else %}
+                <h1>
+                    {% trans "Password Reset" %}
+                </h1>
+
+                <p style="color: rgba(0,0,0,.75);">
+                    {% blocktrans %}You're receiving this e-mail because you requested a password reset for your user account at {{ platform_name }}.{% endblocktrans %}
```

Comment:
_platform_name_ has received a request to reset the password for this account.

If you requested a password reset, click the following link, or paste the URL in your browser's address bar.

Change my password

If you did not request a password reset, you can ignore this email. _platform_name_ has taken no action.

Thanks!
The _platform_name_ team

(note "team" isn't capitalized)